pkg/chains/objects/objects.go

```diff
@@ -135,8 +135,8 @@ func (tro *TaskRunObject) GetPullSecrets() []string {
 type PipelineRunObject struct {
 	// The base PipelineRun
 	*v1beta1.PipelineRun
-	// taskRuns that were apart of this PipelineRun
-	taskRuns []*v1beta1.TaskRun
+	// TaskRuns that were apart of this PipelineRun
+	TaskRuns []*v1beta1.TaskRun
```

Comment:
I think we should avoid this. If we want `TaskRuns` to be accessible, let's make that a method on the interface.

Reply:
SGTM!

Reply:
Done! PTAL https://github.com/tektoncd/chains/pull/879